packages/tangle/message.go

```diff
@@ -40,11 +40,11 @@ const (
 	// MaxParentsCount defines the maximum number of parents each parents block must have.
 	MaxParentsCount = 8
 
-	// defines the minimum number of parents each parents block must have.
+	// MinParentsBlocksCount defines the minimum number of parents each parents block must have.
 	MinParentsBlocksCount = 1
 
-	// defines the maximum number of parents each parents block must have.
-	MaxParentsBlocksCount = 4
+	// MaxParentsBlocksCount defines the maximum number of parents each parents block must have.
+	MaxParentsBlocksCount = 8
```

Comment:
I think this variable should be renamed. Isn't it duplicated with `MaxParentsCount`?

Reply:
seems like it...

Git blame tells me you added it (or modified it last) :detective: 
you want me to fix this as part of this PR?

Reply:
I think we should get rid of all unused and badly-named vars yeah! Especially if I wrote them :P